skills/inferskills.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import json
+
+
+class InferSkills:
+  def __init__(self):
+    self.vals = {}
+    with open('skillsets_labels.tsv', 'r') as f:
+      for line in f:
+        items = line.split('\t')
+        k = items[1]
+        v = set(items[2].replace('\n', '').lower().split(', '))
+        self.vals[k] = v
+  
+  def render_scores(self, data, scores, keywords):
+    for i in data:
+      for k in self.vals.keys():
+        for v in self.vals[k]:
```

Comment:
Consider whether there might be significant performance improvements through sets and set operations rather than checking every field individually. Maybe you should be breaking early if you find a keyword, since I don't know how helpful the 'score' is in practice. Can you take a look at the score variable and see how it does? I'd expect anything caught in 'roles' to get multiplied several times because many of the fields are copies or substrings of one another...